device_test.go

```diff
@@ -156,6 +175,36 @@ func TestClaimAndRelease(t *testing.T) {
 	}
 }
 
+func TestInterfaceDescriptionError(t *testing.T) {
```

Comment:
Can this test function be marked t.Parallel()? If not, add a comment explaining why not.

Reply:
I can't be marked parallel as-is, as newFakeLibusb modifies the shared state (variable libusb). It could be made to run in parallel if the fake libusb was intialized in main rather than per test, but it would still require some changes to the library. I don't think it provides a lot of value now, but we could consider it later if you think it would be beneficial.

Adding comments.